rust_src/src/fileio.rs

```diff
@@ -0,0 +1,18 @@
+//! Functions to deal with files
+
+use lisp::defsubr;
```

Comment:
Please be sure to read our contributing documentation. Your imports do not match our style.

Reply:
Reordered as per the guidelines.